extlinks/programs/templates/programs/program_detail.html

```diff
@@ -44,15 +44,25 @@ <h1>{{ object }}</h1>
                             </div>
                         </div>
                         <div class="form-row">
-                            {% if program.any_orgs_user_list %}
-                                <div class="fieldWrapper" style="padding: 15px;">
-                                    {{ form.limit_to_user_list.errors }}
-                                    {{ form.limit_to_user_list.label_tag }} {{ form.limit_to_user_list }}
-                                    {% if form.limit_to_user_list.help_text %}
-                                        <p class="help">{{ form.limit_to_user_list.help_text|safe }}</p>
-                                    {% endif %}
-                                </div>
-                            {% endif %}
+                            <div class="fieldWrapper" style="padding: 15px;">
+                                {{ form.limit_to_user_list.errors }}
+                                {{ form.limit_to_user_list.label_tag }} {{ form.limit_to_user_list }}
+                                {% if form.limit_to_user_list.help_text %}
+                                    <p class="help">{{ form.limit_to_user_list.help_text|safe }}</p>
+                                {% endif %}
+                            </div>
+                            <div class="col">
+                                <button type="submit" class="btn btn-primary">Submit</button>
+                            </div>
+                        </div>
```

Comment:
This didn't need changing, `{% if program.any_orgs_user_list %}` should remain for the user list checkbox.